### PR TITLE
CAKE-4855 | bump content-types library and add theming

### PR DESCRIPTION
**Pull request by vforge**
@Wikia/cake

### CODE REVIEW

File: extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php [hunk 1]
```diff
@@ -1,24 +1,25 @@
 <?php
-$wgExtensionCredits['other'][] = [
+$wgExtensionCredits[ 'other' ][] = [
```
Comment (mklucsarits) [inline extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php]: We don't use spaces inside arrays in PHP, only inside parentheses.

Comment (mklucsarits) [reply extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php]: I should clarify, when assigning to an existing array, the spaces aren't used. I've looked in other Wikia extensions and it's the same.

Comment (vforge) [reply extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php]: I've seen both in different files (especially `.setup.php` are very inconsistent). I went with your suggestion.

File: extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
 		}
 
+		wfProfileOut( __METHOD__ );
+		return true;
+	}
+
+	public static function onWikiaSkinTopScripts( Array &$vars, &$scripts ) {
+		wfProfileIn( __METHOD__ );
+		$td = (new ThemeSettings())->getSettings();
+		$vars['wgTriviaQuizzesPrimaryColor'] = $td['color-community-header'];
```
Comment (mklucsarits) [inline extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php:19]: This is probably fine, but I'm wondering if there's another naming syntax for these kinds of variables that aren't really wikia global (`$wgVariable`) types but are sort of faked for the front-end. It probably doesn't matter though.

**Review by mklucsarits — APPROVED** [review]

**Pull request merged**